code/game/area/Space_Station_13_areas.dm

```diff
@@ -618,6 +618,11 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	lighting_colour_bulb = "#ffebc1"
 	sound_environment = SOUND_AREA_WOODFLOOR
 
+/area/crew_quarters/bar/mood_check(mob/living/carbon/human/subject)
+		. = ..()
+		if (!HAS_TRAIT(subject, TRAIT_LIGHT_DRINKER))
```

Comment:
```suggestion
		if (HAS_TRAIT(subject, TRAIT_LIGHT_DRINKER))
```

Reply:
Note sure why it committed the unedited version, should be if they do have the trait then it returns false since light drinkers don't get the positive trait.